### PR TITLE
Update issue link.

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
I can't access http://issue.k8s.io/6667. 
https://github.com/kubernetes/kubernetes/issues/6667 is OK

### CODE REVIEW

**Comment by cblecker** [comment]
http://issue.k8s.io/6667 works just fine for me

**Comment by zhangxiaoyu-zidif** [comment]
@cblecker 
en, That may re related with China Mainland net env.
I use VPN, I can access http://issue.k8s.io/6667;
If I don't use VPN, I can not reach http://issue.k8s.io/6667 and other websites, such as Facebook, google, twitter...

**Comment by cblecker** [comment]
@zhangxiaoyu-zidif Does it make any difference if you use https on that URL?

**Comment by zhangxiaoyu-zidif** [comment]
@cblecker 
use https://issue.k8s.io/6667 is good. =)

**Comment by zhangxiaoyu-zidif** [comment]
@cblecker 
Good. Thanks~

File: contributors/devel/local-cluster/local.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ $ tail -f /tmp/kube-apiserver.log
 
 #### The pods fail to connect to the services by host names
 
-The local-up-cluster.sh script doesn't start a DNS service. Similar situation can be found [here](http://issue.k8s.io/6667). You can start a manually.
+The local-up-cluster.sh script doesn't start a DNS service. Similar situation can be found [here](https://github.com/kubernetes/kubernetes/issues/6667). You can start a manually.
```
Comment (cblecker) [inline contributors/devel/local-cluster/local.md]: Do you want to update your PR then to point to https://issue.k8s.io/6667 ?

**Comment by zhangxiaoyu-zidif** [comment]
@cblecker 
Sir, fixed, thanks.

**Pull request merged**